tree/30_generic_methods/log_rudder.cf

```diff
@@ -277,7 +277,8 @@ bundle agent _rudder_common_report(technique_name, status, identifier, component
 
   reports:
     !changes_only|send_reports::
-      "@@${technique_name}@@${status}@@${identifier}@@${component_name}@@${component_key}@@${g.execRun}##${g.uuid}@#${message}";
+      "@@${technique_name}@@${status}@@${identifier}@@${component_name}@@${component_key}@@${g.execRun}##${g.uuid}@#${message}"
+      comment => "${join}";
```

Comment:
Can you improve the text by saying: "Method call with parameters ${join}", or something equivalent ?